### ISSUE TITLE
Windows newline not handled correctly at generation

### ISSUE THREAD

**Issue by waliens** [LucasBassetti/i18n-csv2json-cli#2]
It seems that the client does not handle correctly windows newline `\r\n`.
Indeed for this translations file:

```csv
key,en,fr
and,and,
or,or,

```

The file generated file for french is named `fr.i18n.json` (square being the `\r`)

```json
{
  "and": "\r",
  "or": "\r"
}
```

**Comment by AustinGil** [comment]
Yeah, I just tried getting someone setup on Windows with this and it's not working.
